docs/release-notes/release-notes.md

```diff
@@ -65,7 +65,7 @@ make configure-v3 CONFIG_FILE=path/to/other/config.toml
 The following command can be used, if you are a validator in the active set, to signal to upgrade to v3:
 
 ```bash
-celestia-appd tx signal signal 3 <plus transaction flags>
+celestia-appd tx signal 3 <plus transaction flags>
```

Comment:
see our drive by PR policy in contibution guidelines

this is clearly ai slop that failed the captcha of not deleting a valid sub command